Servers/UI/OJS.Servers.Ui/ClientApp/src/components/problems/problem-resource/ProblemResource.tsx

```diff
@@ -21,35 +21,37 @@ const ProblemResource = ({ resource }: IProblemResourceProps) => {
     const {
         actions: {
             getProblemResourceFile,
-            getProblemResourceResponse,
+            problemResourceContent,
+            clearFileContentState,
         },
     } = useHomeContests();
```

Comment:
Should these be in `useHomeContests`? Is this the `HomePage`?

Reply:
Thats a weird change in [https://github.com/SoftUni-Internal/OpenJudgeSystem/commit/e14d4725](url). I'll refactor it back in the proper hook.

Reply:
It's not a weird commit. Just renamed.